Core/include/Acts/Seeding/SeedFinder.ipp

```diff
@@ -360,16 +360,25 @@ SeedFinder<external_spacepoint_t, platform_t>::getCompatibleDoublets(
       // interactionPointCut == true we apply this cut first cuts before
       // coordinate transformation to avoid unnecessary calculations
       if (std::abs(rM * yNewFrame) <= impactMax * xNewFrame) {
+        const float iDeltaR = std::sqrt(iDeltaR2);
+        const float cotTheta = deltaZ * iDeltaR;
+
         // check if duplet cotTheta is within the region of interest
-        // cotTheta is defined as (deltaZ / deltaR) but instead we multiply
-        // cotThetaMax by deltaR to avoid division
-        if (deltaZ > m_config.cotThetaMax * deltaR or
-            deltaZ < -m_config.cotThetaMax * deltaR) {
+        if (cotTheta > m_config.cotThetaMax or
+            cotTheta < -m_config.cotThetaMax) {
           continue;
         }
 
-        const float iDeltaR = std::sqrt(iDeltaR2);
-        const float cotTheta = deltaZ * iDeltaR;
+        // cut on bottom SPs in a certain (r, eta) region of the detector for
+        // fast seeding
+        if constexpr (candidateType == Acts::SpacePointCandidateType::eBottom) {
+          if (m_config.fastTrackingCut and
```

Comment:
Should we find a better name? Do we envision a situation where we have other tracking cuts? 
Since we are using a delegate we may potentially use a more generic-named flag and then the user will decide the cuts. 

@noemina ?

Reply:
I don't like the use of `fastTracking` here. I agree we need to use a generic name and allow the user defining them. One reason I think we need to work with delegates, is that I would really like to see the effect of these cuts in physics performance and CPU... I would strongly avoid the use of cuts "just because".